.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts
+        with:
+          name: tauri-web
+          path: dist/*
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions-rs/toolchain@v1
+        name: Setup Rust
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Install tauri dependencies
+        run: sudo apt update && sudo apt install -y build-essential libpython3-dev libdbus-1-dev libgtk-3-dev libsoup-gnome2.4-dev libwebkit2gtk-4.0-dev
+      - name: Build Tauri
+        run: yarn run build:tauri
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts (appimage)
+        with:
+          name: tauri-linux
+          path: src-tauri/target/release/bundle/appimage/*.AppImage
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts (deb)
+        with:
+          name: tauri-debian
+          path: src-tauri/target/release/bundle/deb/*.deb
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions-rs/toolchain@v1
+        name: Setup Rust
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build Tauri
+        run: yarn run build:tauri
+      - uses: actions/upload-artifact@v2
+        name: Upload Artifacts
+        with:
+          name: tauri-windows
+          path: src-tauri/target/release/bundle/msi/*.msi
+  build-macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions-rs/toolchain@v1
+        name: Setup Rust
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build Tauri
+        run: yarn run build:tauri
+      - name: Compress App
+        run: tar -cvzf qcvis.app.tar.gz ./src-tauri/target/release/bundle/macos/qcvis.app
+      - uses: actions/upload-artifact@v2
+        name: Upload Artifacts
+        with:
+          name: tauri-macos-dmg
+          path: src-tauri/target/release/bundle/dmg/*.dmg
+      - uses: actions/upload-artifact@v2
+        name: Upload Artifacts
+        with:
+          name: tauri-macos-app
+          path: qcvis.app.tar.gz
+  release:
```

Comment:
There are a few things we need to clear up before we can do automated releases, so I'd prefer to defer this for now

Reply:
This is only a development (pre-)release, releasing the latest trunk, overriding the release each time. This is not planned not supposed to be a tagged release.

Reply:
I know, but there are still some issues we need to clear up first before we do pre-bundled releases (no matter if pre-release or tagged), so I'd like to see this as a separate PR

Reply:
Okay, I will deactivate the release task then.

What kind of blocking issues are there?

Reply:
Mostly regarding the documentation of dependencies in the released versions. I have someone working on it

Reply:
Not 100% happy with our solution (#4) yet, but it'll do for now